resources/config-example/checksums.txt

```diff
@@ -0,0 +1,3 @@
+31a5908bfc16c3cc667ef7940b8b60a7557facc24b9e2023c7276793b121833c  election-manifest.json
```

Comment:
Maybe I missed it, but these are not checked by the server when it starts right? It would be ideal if they were.

Does not have to be now, but can you make a task to do it if time permits. That will give us a bit of safety so we know we haven't screwed up the configs.

Reply:
I only added these checksum files to make it easier for others to construct the parameter for initialising the election contract.

We should check that the files used in the server have the same checksum as the one registered in the contract though. I think it makes sense to do this as this PR for completeness, so will add this.

Reply:
As discussed in the office, the second paragraph is exactly what I had in mind. I just put the comment here because I did not know where else it would fit :)

So we're in agreement.

Reply:
Added verification of the configuration files used by the indexing service by checking the checksums registered in the contract.